spanner/api/Spanner.Samples.Tests/CastDataTypesAsyncPostgreTest.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Threading.Tasks;
+using Xunit;
+
+[Collection(nameof(SpannerFixture))]
+public class CastDataTypesAsyncPostgreTest
+{
+    private readonly SpannerFixture _spannerFixture;
+
+    private readonly CastDataTypesAsyncPostgreSample _sample;
+
+    public CastDataTypesAsyncPostgreTest(SpannerFixture spannerFixture)
+    {
+        _spannerFixture = spannerFixture;
+        _sample = new CastDataTypesAsyncPostgreSample();
+    }
+
+    [Fact]
+    public async Task TestCastDataTypesAsyncPostgre()
+    {
+        // Act.
+        var result = await _sample.CastDataTypesAsyncPostgre(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.PostgreSqlDatabaseId);
+        
+        //Assert.
+        Assert.Contains("String : 1", result);
```

Comment:
Yep, avoid these tests on a constructed string.
Either return a struct from your tests or a value, like the amount of records fetched, etc.